index.html

```diff
@@ -1,177 +1,264 @@
+      <p class="font-montesori hide">
+        A daily selection of privately personalized reads; no accounts or
+        sign-upsrequired. has been the industry's standard dummy text ever
+        since the 1500s, when an unknown printer tooka standard dummy text.
+      </p>
+      <ul class="tools-container hide">
+        <li class="tools bg-btn">html</li>
+        <li class="tools bg-btn">css</li>
+        <li class="tools bg-btn">bootstrap</li>
+        <li class="tools bg-btn">Ruby</li>
+      </ul>
+      <button class="see-project" type="button">See Project</button>
+    </div>
   </section>
   <section id="about-me-container">
-  <div class="about-me-block">
-  <h3 class="works-title yelow abt-me">About me</h3>
-  <p class="font-montesori about-me-p white ">Hello I’m a software developer! I can help you build a product , feature orwebsite Look through some of my work and experience! If you like what you see and have a project youneed coded, don’t hestiate to contact me.</p>
-  <button class="get-resume-btn" type="button">Get My Resume</button>
-  </div>
-  <hr class="line line2">
-  <ul class="categories-container font-montesori white">
-  <li class="cat-title-1-1">Language</li>
-  <ul class="cat-title-2 grid1">
-  <li>Javascript</li>
-  <li>Ruby</li>
-  <li>Html</li>
-  <li>Css</li>
-  </ul>
-  <li class="cat-title-1-2">Frameworks</li>
-  <ul class="cat-title-2 grid2">
-  <li>Bootstrap</li>
-  <li>Ruby on Rails</li>
-  <li>RSPec</li>
-  <li>CapyBara</li>
-  <li>Selenium</li>
-  </ul>
-  <li class="cat-title-1-3">Skills</li>
-  <ul class="cat-title-2 grid3">
-  <li>Codekit</li>
-  <li>Github</li>
-  <li>Codepen</li>
-  <li>Gitlab</li>
-  <li>Terminal</li>
-  </ul>
-  </ul>
+    <div class="about-me-block">
+      <h3 class="works-title yelow abt-me">About me</h3>
+      <p class="font-montesori about-me-p white">
+        Hello I’m a software developer! I can help you build a product ,
+        feature orwebsite Look through some of my work and experience! If you
+        like what you see and have a project youneed coded, don’t hestiate to
+        contact me.
+      </p>
+      <button class="get-resume-btn" type="button">Get My Resume</button>
+    </div>
+    <hr class="line line2" />
+    <ul class="categories-container font-montesori white">
+      <li class="cat-title-1-1">Language</li>
+      <ul class="cat-title-2 grid1">
+        <li>Javascript</li>
+        <li>Ruby</li>
+        <li>Html</li>
+        <li>Css</li>
+      </ul>
+      <li class="cat-title-1-2">Frameworks</li>
+      <ul class="cat-title-2 grid2">
+        <li>Bootstrap</li>
+        <li>Ruby on Rails</li>
+        <li>RSPec</li>
+        <li>CapyBara</li>
+        <li>Selenium</li>
+      </ul>
+      <li class="cat-title-1-3">Skills</li>
+      <ul class="cat-title-2 grid3">
+        <li>Codekit</li>
+        <li>Github</li>
+        <li>Codepen</li>
+        <li>Gitlab</li>
+        <li>Terminal</li>
+      </ul>
+    </ul>
```

Comment:
♻️ Well done using lists to structure your skills/stack! However, as part of the accessibility checks, the basic structure of the page should be navigable. Please note how a `ul` should contain `li` items only. That way, the screen readers would be able to navigate the page correctly.

![image](https://user-images.githubusercontent.com/26911718/207357722-955b4dcf-ea66-41ef-a578-60b290892f9c.png)

- [ ] Kidnly update your list so the inner list (the inner `ul` tag) is placed inside another `li` tag. That is, the inner list should be another item of the outer list.